jenkins/ci.suse.de/pipelines/openstack-ardana-testbuild-gerrit.Jenkinsfile

```diff
@@ -32,7 +32,7 @@ pipeline {
             returnStdout: true,
             script: '''
               gerrit_change_ids=$(echo $gerrit_change_ids|tr ",/" -)
-              echo http://download.suse.de/ibs/${homeproject//:/:\\/}:/ardana-ci-${gerrit_change_ids}/standard/${homeproject}:ardana-ci-${gerrit_change_ids}.repo
+              echo http://download.suse.de/ibs/${homeproject//:/:\\/}:/ardana-ci-${gerrit_change_ids}/standard/${homeproject}:ardana-ci-build-${BUILD_NUMBER}-changes-${gerrit_change_ids}.repo
```

Comment:
this also needs to be updated to reflect the new IBS project/repository name

Reply:
Ah yes, thanks.